docs/ReleaseNotes.md

```diff
@@ -40,6 +40,7 @@ The `IndexEntry` class now contains a reference to its source index. This meant
 
 The API stability annotations have been moved to their own package. This allows for references to the stability levels within Javadoc pages to link to the annotation documentation. Any external users who had also used these annotations will be required to modify their imports to match the new package. Because these annotations have `CLASS` retention level, this change also requires that any client who depends on the Record Layer update any transitive dependency on `fdb-extensions` to 2.6 to avoid compile-time warnings about missing annotation classes.
 
+The `IndexMaintainer` class now has a new public method `validateEntries`. Subclasses inheriting it should implement the new method. It does not break classes inheriting `StandardIndexMaintainer` which has a default implementation that does nothing.
```

Comment:
Needs a newline after this text.

Reply:
I think also maybe "inheriting" to "extending"?

Reply:
Done.